src/StaticCaching/ServiceProvider.php

```diff
@@ -91,6 +91,14 @@ public function boot()
             return app(Cacher::class)->getUrl($this);
         });
 
+        Response::macro('makeCacheControlCacheable', function () {
```

Comment:
Looks like there's a missing import here, causing the test suite to fail. Can you fix it?

Reply:
Fixed, during development i had it in the ResponseStatusTracker which was not the right place in the end but had the import 😅